libs/community/langchain_community/vectorstores/azure_cosmos_db_no_sql.py

```diff
@@ -274,38 +313,40 @@ def _similarity_search_with_score(
             query += "TOP @limit "
 
         query += (
-            "c.id, c.{}, c.text, c.metadata, "
-            "VectorDistance(c.@embeddingKey, @embeddings) AS SimilarityScore FROM c"
+            "c.id, c.text, c.metadata, c.embedding, "
+            "VectorDistance(c.embedding, @embeddings) AS SimilarityScore FROM c"
         )
 
         # Add where_clause if specified
         if pre_filter is not None and pre_filter.get("where_clause") is not None:
             query += " {}".format(pre_filter["where_clause"])
 
-        query += " ORDER BY VectorDistance(c.@embeddingKey, @embeddings)"
+        query += " ORDER BY VectorDistance(c.embedding, @embeddings)"
```

Comment:
Here too: is this a breaking change, and if so, what is the motivation?

Reply:
This is not a breaking change. Sometimes the @embeddingKey is not working in the query and throwing a BadRequestException sometimes. And we are having it as c.embedding in our CosmosDB SDK as well